src/auth/ejabberd_auth_odbc.erl

```diff
@@ -128,11 +128,10 @@ check_password(LUser, LServer, Password, Digest, DigestGen) ->
                        "reason=~p user=~ts", [Error, LUser]),
             false %% Typical error is that table doesn't exist
     catch
-        Class:Reason ->
-            Stacktrace = erlang:get_stacktrace(),
+        ?EXCEPTION(Class, Reason, CURRENT_STACKTRACE) ->
```

Comment:
Can we make it just `Stacktrace`? After preprocessing it's an ordinary variable after all.